_data/scout-cli/docker_scout_compare.yaml

```diff
@@ -0,0 +1,212 @@
+command: docker scout compare
+short: '[early preview] Compare two images and display differences'
+long: |-
+    The `docker scout compare` command analyzes two images and display a comparison of both.
+
+    > This command is in **early preview** and its behaviour might change in the future
+
+    The main usage is to compare two versions of the same image.
+    For instance when a new image is built and compared to the version running in production.
+
+    The following artifact types are supported:
+
+    - Images
+    - OCI layout directories
+    - Tarball archives, as created by `docker save`
+
+    The tool analyzes the provided software artifact, and generates a vulnerability report.
+
+    By default, the tool expects an image reference, such as:
```

Comment:
```suggestion
    By default, the command expects an image reference, such as:
```

Reply:
Not sure about this one, or we need to change it in all other commands (this part is just a copy/paste of the others)